.travis/script.sh

```diff
@@ -154,6 +154,32 @@ EOF
         sudo ./unmount.sh $IMAGE_DIR) || die
     ;;
 
+docker-check)
+    docker build -t otbr -f etc/Dockerfile . || die
```

Comment:
It will be better to make this check locally runnable. See https://github.com/openthread/borderrouter/pull/205